### ISSUE TITLE
RubyTime object is not Logstash::Timestamp

### ISSUE THREAD

**Issue by untergeek** [logstash-plugins/logstash-input-jdbc#77]
It appears that the JDBC plugin is grabbing timestamps correctly, but it's only converting them to `RubyTime` objects, which is not the same thing as a `Logstash::Timestamp`.  This conversion should be possible without resorting to a `ruby{}` filter.

See: https://discuss.elastic.co/t/logstash-date-filter-question/33663

**Comment by untergeek** [comment]
A temporary workaround:

```
ruby {
  code => "event['event_date'] = LogStash::Timestamp.new(event['event_date'])"
}
```

**Comment by CraigFoote** [comment]
Bad news Aaron, I'm getting an error with your workaround:
```
Ruby exception occurred: uninitialized constant Logstash::Filters::Ruby::Logstash {:level =>:error, :file=>"logstash/filters/ruby.rb", :line=>"41", :method=>"filter"}
```

**Comment by CraigFoote** [comment]
My apologies, I transcribed "LogStash::Timestamp.new" as "Logstash::Timestamp.new". When I capitalized the "S" your code worked. Thanks again.

**Comment by untergeek** [comment]
Excellent!  I'll close this now.

**Comment by CraigFoote** [comment]
I meant your workaround code works. The jdbc input plugin is still indexing date fields as strings.

**Comment by untergeek** [comment]
Right. Oops :smile:

**Comment by suyograo** [comment]
This has been fixed in #83
